libcfnet/file_stream.c

```diff
@@ -829,6 +836,20 @@ static bool SendSignature(SSL *conn, const char *filename)
     fclose(file);
     rs_job_free(job);
 
+    const char *msg =
+        "Send signature statistics:\n"
+        "  %zu bytes in (read from '%s')\n"
+        "  %zu bytes out (sent to server)\n";
+
+    if (print_stats)
+    {
+        fprintf(stderr, msg, bytes_in, filename, bytes_out);
+    }
+    else
```

Comment:
I wonder if we should always log and only optionally print to stderr. Probably doesn't matter much but seems more right to me somehow.

Reply:
Either way is fine by me :+1: